LayoutTests/media/wireless-playback-media-player/timeupdate-without-position-updates.html

```
@@ -0,0 +1,46 @@
+<!DOCTYPE html><!-- webkit-test-runner [ WirelessPlaybackMediaPlayerEnabled=true ] -->
+<html>
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)